pkg/daemon/ceph/client/filesystem.go

```diff
@@ -121,11 +121,15 @@ func AllowStandbyReplay(context *clusterd.Context, clusterName string, fsName st
 }
 
 // CreateFilesystem performs software configuration steps for Ceph to provide a new filesystem.
-func CreateFilesystem(context *clusterd.Context, clusterName, name, metadataPool string, dataPools []string) error {
+func CreateFilesystem(context *clusterd.Context, clusterName, name, metadataPool string, dataPools []string, force bool) error {
 	if len(dataPools) == 0 {
 		return fmt.Errorf("at least one data pool is required")
 	}
 
+	if force {
+		logger.Infof("Filesystem %s will reuse pre-existing pools", name)
```

Comment:
would you mind moving this down to line 150? there is already a check for `force` there.

Reply:
Done. Thanks!